lib/game.rb

```diff
@@ -21,12 +21,12 @@ def lowest_total_score
 
   #Work in Progress
   def count_of_games_by_season
-    season_hash = {}
+    season_hash = Hash.new(0)
```

Comment:
We have to be mindful of making sure the default is zero when we add to a key value pair. I appreciate it how you established this default for the success of the method.